lib/main.js

```diff
@@ -54,18 +53,18 @@ function weave(format) {
     if (process.platform === 'win32') {
         edpath = edpath.replace(/\\/g, "/")
     }
-    outpath = edpath.split('.')
+    let outpath = edpath.split('.')
     outpath.pop()
     outpath = outpath.join('.') + '.' + format
 
-    weaveCommand  = 'using Weave;'
```

Comment:
Pretty sure we should keep the `using Weave` part around, otherwise the UX will be bad, imho.

Reply:
Ah sorry for this, I just forget to add this cause I included `using Weave` in juno_startup.jl.
Fix this soon.